wrapanapi/systems/rhevm.py

```diff
@@ -1005,10 +1005,10 @@ def _get_image_service(self, storage_domain_name, image_name):
 
     def import_glance_image(self, source_storage_domain_name, source_template_name,
                             target_storage_domain_name, target_cluster_name, target_template_name,
-                            async=True, import_as_template=True):
```

Comment:
it might be better off using whole word.

Reply:
I am just trying to fallow `oVirt` here.